wagtail/models/__init__.py

```diff
@@ -399,6 +399,7 @@ def save_revision(
             user=user,
             approved_go_live_at=approved_go_live_at,
             content=self.serializable_data(),
+            title=getattr(self, "title", str(self)),
```

Comment:
The [``__str__`` method on Page](https://github.com/wagtail/wagtail/blob/73a79fd394822edb57367dc8f3af4ae617d45520/wagtail/models/__init__.py#L404) returns the title so perhaps we only need ``str(self)``?

Reply:
What if the user overrides `__str__()` in their specific page model (since `save_revision()` can only be called from the specific instance)?

Reply:
@gasman 

We're thinking, to keep pages consistent with Snippets, that we change this title field to always do ``str(object)`` and not look for a ``title`` field. This changes the behaviour slightly as if someone has overridden an ``__str__`` method on a page model, the draft title shown in the admin would use that instead of the title.

Do you think this is OK?

Reply:
@kaedroho Yeah, that seems fine to me - I can't think of any reason why you'd override `__str__` and _not_ expect the result to surface in situations like this.